core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -137,10 +137,38 @@ class CleanerTest extends JUnitSuite {
                (0 until leo.toInt by 2).forall(!keys.contains(_)))
   }
 
+  def testLogCleanerStats(): Unit = {
```

Comment:
@ijuma It is a bit awkward. We need to either work through `LogCleaner` which would make the test asynchronous, or we need to expose `CleanerThread` which is nested in `LogCleaner`. It would be nice to get some more coverage in here, but I'm sort of inclined to leave this for future work.